src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs

```diff
@@ -519,7 +521,7 @@ private static Value ConvertToPrimitiveValue(ProtoValue value)
 
         private Service.ServiceClient BuildClientForPlatform(Uri url)
         {
-            var useUnixSocket = url.ToString().StartsWith("unix://");
+            var useUnixSocket = url.Scheme is "unix";
```

Comment:
Above, you use `string.Equals(url.Scheme, "unix", StringComparison.OrdinalIgnoreCase)`. Is there a reason to do it differently here?

Reply:
Not necessarily. I'm starting to see the `is {const string}` pattern more often in the wild for exact/ordinal matching, so since `StartsWith(string)` implies `StartsWith(string, StringComparison.Ordinal)`, I converted it to a constant ordinal pattern out of habit.

But I can certainly convert it to `string.Equals(url.Scheme, "unix", StringComparison.Ordinal)` for consistency if you prefer.

Reply:
I have a weak preference for the consistency in this case.

Reply:
Fair enough! Updated:

```diff
diff --git a/src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs b/src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs
index 6496ebc..cdb5f26 100644
--- a/src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs
+++ b/src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs
@@ -516,7 +516,7 @@ namespace OpenFeature.Contrib.Providers.Flagd

         private Service.ServiceClient BuildClientForPlatform(Uri url)
         {
-            var useUnixSocket = url.Scheme is "unix";
+            var useUnixSocket = string.Equals(url.Scheme, "unix", StringComparison.Ordinal);

             if (!useUnixSocket)
             {
```